app/controllers/event_participations_controller.rb

```diff
@@ -0,0 +1,29 @@
+class EventParticipationsController < ApplicationController
+  include ApplicationHelper
+  before_filter :authenticate_user!
+
+  def create
+    intention = params[:event_participation_intention]
+    event_participation = current_user.participate_in_event!(target, intention) if target
+    respond_to do |format|
+      format.html { redirect_to :back }
+      format.mobile { redirect_to stream_path }
+      format.json { render json: event_participation, :status => 201 }
+    end
+    rescue ActiveRecord::RecordInvalid
```

Comment:
Indent the rescue to the same level as its opening statement (in this case `def`).